geotrek/signage/serializers.py

```diff
@@ -20,11 +20,17 @@ class Meta:
         fields = ('id', 'pictogram', 'label')
 
 
+class ConditionsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = signage_models.SignageCondition
+        fields = ('label',)
+
+
 class SignageSerializer(DynamicFieldsMixin, BasePublishableSerializerMixin, serializers.ModelSerializer):
     name = serializers.CharField(source='name_display')
     structure = serializers.SlugRelatedField('name', read_only=True)
     type = serializers.CharField(source='type_display')
-    condition = serializers.SlugRelatedField('label', read_only=True)
+    conditions = serializers.CharField(source='conditions_display')
```

Comment:
Is `ConditionsSerializer` class used anywhere, since we're using `conditions_display` here ?